lib/testing/src/lib/core/pages/form/widgets/attachFileWidgetCloud.ts

```diff
@@ -30,8 +30,9 @@ export class AttachFileWidgetCloud {
         this.assignWidget(fieldId);
     }
 
-    assignWidget(fieldId: string): void {
-        this.widget = this.formFields.getWidget(fieldId);
+    async assignWidget(fieldId: string): Promise<void> {
+        this.widget = await this.formFields.getWidget(fieldId);
```

Comment:
AFAIK, getWidget returns an ElementFinder. If that's the case, await should be removed